### PR TITLE
Connection Status

### PR DESCRIPTION
**Pull request by nishikawa7863**

### CODE REVIEW

File: src/components/dash/header.tsx [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
 import {useStores} from '../../store'
 import { useObserver } from 'mobx-react-lite'
 
 export default function Header() {
   const navigation = useNavigation()
-  const {details} = useStores()
+  const {details, ui} = useStores()
+
+  const checkConnectionHandler = () => {
+    const connected = ui.connected ? 'Connected node' : 'Disconnected noe';
```
Comment (pitoi) [inline src/components/dash/header.tsx:14]: noe -> node

Comment (nishikawa7863) [reply src/components/dash/header.tsx]: @pitoi sorry about that! should be fixed now!

**Comment by Evanfeenstra** [comment]
For the life of me I cant get these dependencies to build, keep running into Error: "Unable to resolve module `./toolbar-android` from `node_modules/react-native-vector-icons/lib/create-icon-set.js`"

**Comment by nishikawa7863** [comment]
@Evanfeenstra can you share your env details? Node version, OS? I'll try to replicate and provide a fix.

**Pull request closed without merging**